docs/reference/target-group-policy.md

```diff
@@ -53,7 +55,15 @@ Updates to this configuration result in a replacement of VPC Lattice TargetGroup
 
 Appears on: TargetGroupPolicySpec
 
-HealthCheckConfig defines health check configuration for given VPC Lattice target group. For the detailed explanation and supported values, please refer to [VPC Lattice documentation](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) on health checks.
+HealthCheckConfig defines health check configuration for given VPC Lattice target group.
+
+For the detailed explanation and supported values, please refer to [VPC Lattice documentation](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) on health checks.
+
+**Limitations and Considerations**
+* Omitting `healthCheck` results in [VPC Lattice default behavior](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) depending on the protocol.
+  * Health check is enabled by default for HTTP1 target groups.
+  * Health check is disabled by default for HTTP2/gRPC target groups.
+* For targets behind GRPCRoute, you should create a separate endpoint dedicated for health checks - HTTP/2 health check directly on gRPC endpoints is not supported.
```

Comment:
> you should create a separate endpoint dedicated for health checks 

may be just copy aws docs? 

> Health checks do not support gRPC target group protocol versions. However, if you enable health checks, you must specify the health check protocol version as HTTP1 or HTTP2.

Reply:
I tried to address ambiguity from aws docs there, there are some cases users trying to put HTTP2 health checks on GRPC targets directly and we want to avoid it.